src/packages/Toast.vue

```diff
@@ -428,10 +432,9 @@ onMounted(() => {
     ]
     emit('update:heights', newHeights)
   }
-  mounted.value = true
 })
 
-onUnmounted(() => {
+onBeforeUnmount(() => {
```

Comment:
Sry forgot to go back to `onUnmounted` after testing for `offset` calculations problem


Will keep it as is, and wait for your review